lib/src/loader/loader.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="{{ className }}" [ngClass]="size | loaderClasses: cover:fullscreen:type:backdrop" [attr.role]="role">
```

Comment:
wouldn't this result in a class named undefined if you don't pass a custom className?